### PR TITLE
Multiple QuickFixes to Giveaways

### PR DESCRIPTION
**Pull request by SerperiorBae**
- Add more user friendly CSS To them
- Will show Card being given away & PM the room to notify.
- Card / Credit enhancement iffy at this time, still requires a few things for take card/give card

### CODE REVIEW

**Comment by AlainDuhHero** [comment]
ye its easy
contact me if u need help

File: chat-plugins/wifi.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
 /**
-* Wi-Fi chat-plugin. Only works in a room with id 'marketplace'
-* Handles giveaways in the formats: question, lottery
-* Credits: Codelegend, SilverTactic, DanielCranham, bumbadadabum
-**/
-
+ * Wi-Fi chat-plugin. Only works in a room with id 'marketplace'
```
Comment (panpawn) [inline chat-plugins/wifi.js:2]: why are we adding trailing whitespace here

File: chat-plugins/wifi.js [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+ * Handles giveaways in the formats: question, lottery
+ * Credits: Codelegend, SilverTactic, DanielCranham, bumbadadabum
+ * Additional Wisp Edits by: Murdering/Sapience ;3;
+ **/
 'use strict';
 
+const Giveaway_Display = 'background-color: #ffc775 ; border: #ff560e solid 1px ; color: #ff560e; padding: 3px; text-align: center';
```
Comment (panpawn) [inline chat-plugins/wifi.js]: all caps variable names with underscores are preferred

**Comment by panpawn** [comment]
squash

**Comment by panpawn** [comment]
squash all of these commits

**Comment by panpawn** [comment]
also, preview of what this css and design looks like?

**Comment by jd4564** [comment]
Merged in 082e4df319cf335008354d16a31f97f19b9d638a

**Pull request closed without merging**